src/pages/workspaces/List.js

```diff
@@ -101,6 +102,9 @@ export const WorkspaceList = () => {
   //A user may have lost access to a workspace after viewing it, so we'll filter those out just in case
   const recentlyViewed = useMemo(() => _.filter(w => _.find({ workspace: { workspaceId: w.workspaceId } }, workspaces), getLocalPref(recentlyViewedPersistenceId)?.recentlyViewed || []), [workspaces])
 
+  const persistenceId = `workspaces/list`
+  const [recentlyViewedOpen, setRecentlyViewedOpen] = useState(() => getLocalPref(persistenceId)?.recentlyViewedOpen)
```

Comment:
Currently, there are two places that handle the case where `recentlyViewedOpen` is undefined: the Collapse component's `initialOpenState` and `onClick`. Alternatively, that could be handled once here.
```suggestion
  const [recentlyViewedOpen, setRecentlyViewedOpen] = useState(() => _.defaultTo(true, getLocalPref(persistenceId)?.recentlyViewedOpen))
```

Reply:
Much cleaner, thank you!